src/github.com/getlantern/flashlight/logging/logging.go

```diff
@@ -183,3 +183,46 @@ func (w logglyErrorWriter) Write(b []byte) (int, error) {
 	}
 	return len(b), nil
 }
+
+type nonStopWriter struct {
+	writers []io.Writer
+}
+
+// NonStopWriter creates a writer that duplicates its writes to all the
+// provided writers, even if errors encountered while writting.
+func NonStopWriter(writers ...io.Writer) io.Writer {
+	w := make([]io.Writer, len(writers))
+	copy(w, writers)
+	return &nonStopWriter{w}
+}
+
+// Write implements the method from io.Writer. It returns the smallest number
+// of bytes written to any of the writers and the first error encountered in
+// writing to any of the writers.
+func (t *nonStopWriter) Write(p []byte) (int, error) {
+	var fn int
```

Comment:
@fffw I added error handling to Write so that it doesn't swallow errors.  It still writes to every output though.